dataactcore/models/jobTrackerInterface.py

```diff
@@ -285,3 +285,63 @@ def getSubmissionStatus(self,submissionId,interfaces):
             else:
                 return "validation_successful"
         return "unknown"
+
+    def getSubmissionById(self, submissionId):
+        """ Return submission object that matches ID"""
+        query = self.session.query(Submission).filter(Submission.submission_id == submissionId)
+        return self.runUniqueQuery(query, "No submission with that ID", "Multiple submissions with that ID")
+
+    def getSubmissionErrorInfo(self, submissionId):
```

Comment:
a minor thing, but should this function be called `setSubmissionErrorInfo` since it's doing an update?